DARTS/darts.py

```diff
@@ -0,0 +1,180 @@
+import torch
+import torch.nn as nn
+import numpy as np
+import os
+import nibabel
+import re
+
+from DARTS.models.dense_unet_model import Single_level_densenet, Down_sample, Upsample_n_Concat, Dense_Unet
+from DARTS.utils import load_data, create_one_hot_seg, back_to_original_4_pred, back_to_original_4_prob, pickling
+from DARTS.models.unet import Unet, Downsample_block, Upsample_block
+
```

Comment:
Notice the imports, I may have structured them properly so modules are present in `DARTS/` and `DARTS/models/` directories.